test/test-functions

```diff
@@ -314,7 +314,8 @@ install_dmevent() {
     if [[ "$LOOKS_LIKE_DEBIAN" ]]; then
         # dmsetup installs 55-dm and 60-persistent-storage-dm on Debian/Ubuntu
         # see https://anonscm.debian.org/cgit/pkg-lvm/lvm2.git/tree/debian/patches/0007-udev.patch
-        inst_rules 55-dm.rules 60-persistent-storage-dm.rules
+        inst_rules 55-dm.rules 55-storage-hardware-rules \
```

Comment:
wrong file name again...

Reply:
upps big sorry. I really tested this, no idea how that slipped in.